jira/src/main/kotlin/gropius/sync/jira/config/IMSConfig.kt

```diff
@@ -15,14 +15,16 @@ import java.net.URI
  * @param imsTemplate the template of the current IMS
  * @param defaultType the type of newly created issues
  * @param defaultTemplate the template of newly created issues
+ * @param isCloud true if cloud, false if data center, null if not specified
```

Comment:
maybe this would be the better place to do the fallback stuff

Reply:
aa17290ba460cdd247243a1003be89b6b403b445